src/lighteval/logging/info_loggers.py

```diff
@@ -207,6 +207,7 @@ class Detail:
         gold_index: list = field(default_factory=list)
         metrics: dict = field(default_factory=dict)
         specifics: dict = field(default_factory=dict)
+        unconditioned_query: Optional[str] = None
```

Comment:
This should go into specifics instead - that's where we put all the task/metric specific added logic